### ISSUE TITLE
During checkout: missing successful global message when user selects different Shipping Address

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13730]
**Describe the bug**
- Screen reader users wouldn't be able to tell if they have switched/selected an additional Shipping Address during checkout.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**

1. Log in as a registered user
2. Add items to the cart and start the checkout process.
3. Open any Screen Reader (VO, NVDA, JAWS).
4. Switch Shipping Address from default to an additional one.
5. Listen to Screen Reader feedback
6. Hear the absence of confirmation a change has happened.

**Expected behavior**
- Screen reader users will hear a successful confirmation message saying they have changed/selected a different Shipping Address. E.g _"Shipping Address selection successfully changed"._
- For `MSG_TYPE_CONFIRMATION` use `aria-live="polite" aria-atomic="true"`

**Screenshots**

Message suggestion in the screenshot below, please validate phrasing with @Manurod2018 or @vekterdude

![change2021-09-15_10-18-21](https://user-images.githubusercontent.com/15113219/133455735-afa9395c-20c9-4603-b0e7-66ce7bc91635.jpg)

**Comment by BenjaminEmiliani** [comment]
NOTE Any Global Message emitted is now read by Screen Readers (recent change), the only change required to solve this ticket is to trigger a Global Message upon switching/selecting an additional Payment Method during checkout.

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.

![ChangeShippingAdd](https://user-images.githubusercontent.com/45409515/151830621-5121d45f-ddf6-435e-bb49-b78f3f3c72b6.png)
